tools/BugReportTool/BugReportTool/BugReportTool.vcxproj

```diff
@@ -45,7 +46,7 @@
   <ItemGroup>
     <ClCompile Include="..\..\..\deps\cziplib\src\zip.c">
       <!-- Disabling warnings for external code -->
-	  <DisableSpecificWarnings>26451;4267;4244;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+	  <DisableSpecificWarnings>4706;26451;4267;4244;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
Why is there a new warning added here?

Reply:
This file is external file from deps being compiled as part of BugReportTool.
As the PR https://github.com/microsoft/PowerToys/pull/21434 enables 4706, it needs to be disable here to compile this external file.

Reply:
This is already changed by the https://github.com/microsoft/PowerToys/pull/21434

Reply:
Got it 👍  Thank you!